Jenkinsfile

```diff
@@ -33,53 +21,8 @@ pipeline {
             // publishCoverage adapters: [jacocoAdapter('coverage.xml')]
           }
         }
-        stage('opensuse42') {
-          agent { label 'opensuse42' }
-          steps { sh readFile(file: 'jenkins/suse-tests.sh') }
-        }
-        // stage('sles11') {
-        //   // FIXME:  sles11 can't be tested due missing python deps (incl. nose)
-        // }
-        stage('sles12') {
-          agent { label 'sles12' }
-          steps { sh readFile(file: 'jenkins/suse-tests.sh') }
-        }
-        // TODO: add after QE creates pipeline
-        // stage('Functional') {
-        //   stages{
-        //     stage('Build RPM') {steps {echo 'Build RPM'}}
-        //     stage('Prepare') {steps {echo 'Prepare'}}
-        //     stage('Provision') {steps {echo 'Provisioning'}}
-        //     stage('Tier 1') {steps {echo 'Tier 1'}}
-        //   }
-        // }
-      }
-    }
-    stage('SUSE Builds') {
```

Comment:
I can see, why building of SLES 11 was removed, but why did you remove all tests related to OpenSUSE?